src/audio/level_multiplier/tune/sof_level_multiplier_blobs.m

```diff
@@ -0,0 +1,71 @@
+% Export configuration blobs for Level Multiplier
+
+% SPDX-License-Identifier: BSD-3-Clause
+%
+% Copyright (c) 2025, Intel Corporation.
```

Comment:
Can you comment on how this script is used ?

Reply:
Yep, I added it. It shows e.g. with "help sof_level_multiplier_blobs" from Octave shell (it prints the first comments chunk from beginning of file).